### DISCUSSION TITLE
无法登录，通过邮箱登录但是收不到验证码

### DISCUSSION THREAD

**Discussion by Sanwan777** [getlantern/lantern-forum-cn#1067] [Q&A] [answered]
无法登录，通过邮箱登录但是收不到验证码

**Comment by MaxMKMax** [comment]
@Sanwan777 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去. 谢谢!

**Comment by Sanwan777** [comment]
***@***.***



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月03日 23:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 无法登录，通过邮箱登录但是收不到验证码 (Discussion #1067) |

@Sanwan777 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去. 谢谢!

**Comment by Sanwan777** [comment]
有结果了么


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月03日 23:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 无法登录，通过邮箱登录但是收不到验证码 (Discussion #1067) |

@Sanwan777 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去. 谢谢!

**Comment by Sanwan777** [comment]
什么进展了，买了账号，一直都不可以用，这个损失你们都不管的吗？ 这么缓慢



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月03日 23:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 无法登录，通过邮箱登录但是收不到验证码 (Discussion #1067) |

@Sanwan777 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去. 谢谢!

**Comment by Sanwan777** [comment]
什么进展了，买了账号，一直都不可以用，这个损失你们都不管的吗？ 这么缓慢







---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年12月03日 23:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 无法登录，通过邮箱登录但是收不到验证码 (Discussion #1067) |

@Sanwan777 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去. 谢谢!

**Comment by MaxMKMax** [comment]
@Sanwan777 您好. 非常抱歉让您久等了! 还没收到您的支付单据和账号邮箱. 所以无法给您提交问题上去处理. 您的账号邮箱被Github屏蔽了. 您的邮箱这样写给我吧. xxxx @ 163 . com

**Comment by Sanwan777** [comment]
我的 账号是  ***@***.***,     你看下是不是这个单据，有点久远，有点分清了。
​








---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2025年01月07日 02:38 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 无法登录，通过邮箱登录但是收不到验证码 (Discussion #1067) |

@Sanwan777 您好. 非常抱歉让您久等了! 还没收到您的支付单据和账号邮箱. 所以无法给您提交问题上去处理. 您的账号邮箱被Github屏蔽了. 您的邮箱这样写给我吧. xxxx @ 163 . com

**Comment by MaxMKMax** [comment, marked as answer]
https://support.lantern.io/en/support/tickets/new
@Sanwan777 您好. 您用以上的链接直接提交问题吧. 很快支付系统团队就会回复您邮件了. 不知道为什么您发的所有信息都被屏蔽了. Thanks!

**Reply by Sanwan777** [reply]
![Uploading mmexport1736411474265.jpg…]()
我的账号邮箱是， sanwansun@163.com

**Reply by Sanwan777** [reply]
![Uploading mmexport1736411474265.jpg…]()
